ghost/api-framework/lib/pipeline.js

```diff
@@ -267,6 +273,10 @@ const pipeline = (apiController, apiUtils, apiType) => {
         Object.assign(obj[method], apiImpl);
         return obj;
     }, {});
+
+    controllerMap.set(apiController, result);
```

Comment:
Would it be worth using `docName` as the key?

Reply:
`docName` isn't unique - so no! 😝

Reply:
wtf

Reply:
One of the things `docName` is used for is to determine which permissions to use - so the `pages` endpoint has a `docName` of `'posts'` - as does the `postsPublic` endpoint

Reply:
unbelievable